experiment/run_experiment.py

```diff
@@ -364,6 +369,10 @@ def start(self):
             '/var/run/docker.sock:/var/run/docker.sock',
             '-v',
             shared_volume_volume_arg,
```

Comment:
Is this still needed?

Reply:
Will test this later.

Reply:
Still needed in `local_build.py`.
https://github.com/google/fuzzbench/blob/master/experiment/build/local_build.py

Reply:
Sure. Will do it in another PR.
Related to https://github.com/google/fuzzbench/issues/457.